client/components/Tasks.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ThemeContext from './ThemeContext';
```

Comment:
Change the context to be called UserContext. Also missing an import for { useContext} how are you using context on line 7 without that import?